packages/css/src/components/form/form-groups/search-bar/form-group-search-bar.template.ts

```diff
@@ -0,0 +1,53 @@
+import { FormGroupSearchBar } from '@dso-toolkit/sources';
+
+import { html, nothing, TemplateResult } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+
+import { infoButtonTemplate } from '../../../info-button/info-button.template';
+import { infoTemplate } from '../../../info/info.template';
+import { searchBarTemplate } from '../../../search-bar/search-bar.template';
+
+export function formGroupSearchBarTemplate(formGroup: FormGroupSearchBar<TemplateResult>): TemplateResult {
+  const errorTextId = `${formGroup.id}-error-text`;
+  const helpTextId = `${formGroup.id}-help-text`;
+  const infoTextId = `${formGroup.id}-info-text`;
+
+  const ariaDescribedBy = [
+    formGroup.info?.fixed ? infoTextId : formGroup.errorText ? errorTextId : formGroup.helpText ? helpTextId : undefined
```

Comment:
Dit is nu een OR-achtige constructie, maar het is mogelijk om er meerdere te hebben. Zie bijvoorbeeld Form Group Input

Reply:
Aha, dit heb ik helemaal verkeerd op gezet. Nu staat het weer goed